config/locales/en.yml

```diff
@@ -1156,6 +1157,7 @@ en:
           platform: "Railway Platform"
           preserved: "Preserved Railway"
           proposed: "Proposed Railway"
+          rail: "Rail"
```

Comment:
would "Railway track" be more accurate?

The page for `railway=rail` does not mention "track" much.

[https://wiki.openstreetmap.org/wiki/Tag:railway%3Drail](https://wiki.openstreetmap.org/wiki/Tag:railway%3Drail)